zkfold-base.cabal

```diff
@@ -342,4 +347,26 @@ benchmark compiler-benchmark
       tasty-bench,
       tasty-golden,
       zkfold-base,
-      zkfold-symbolic-examples
+      zkfold-symbolic-examples,
+      time,
```

Comment:
These dependencies are not needed in `compiler-benchmark`

Reply:
Probably rebase artifacts